Formula/r/ratchet.rb

```diff
@@ -19,14 +20,24 @@ class Ratchet < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w")
+    ldflags = %W[
+      -s
+      -w
+      -X=github.com/sethvargo/ratchet/internal/version.version=#{version}
+      -X=github.com/sethvargo/ratchet/internal/version.commit=#{Utils.git_head}
+      -extldflags=-static
```

Comment:
```suggestion
```
I don't think we really want this (and I doubt it actually changes anything for the build here...)

Reply:
I'm happy to remove it, but this tells Go not to link against shared libraries (building a full static binary).

Reply:
I can see that you'd want to do that as a project distributing your own binaries, but probably not so much if they're binaries distributed via a package manager. (It also doesn't really work as you expect on macOS.)

Reply:
Good to know, thanks. Done!